### PR TITLE
Prepare a new release.

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## [Unreleased]
+Nothing yet.
+
+
+## [0.7.0] - 2020-06-01
+### Added
+- `disable_basic_metrics` and `disable_enhanced_metrics` configuration options.
+
+### Changed
+- Major changes in `node_exporter`-like metrics.
```
Comment (BupycHuk) [inline CHANGELOG.md:18]: I think it's good to add a link what's changed in these metrics(naming and e.t.c.)

Comment (AlekSi) [reply CHANGELOG.md:18]: Sure, what link should be used there?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## [Unreleased]
+Nothing yet.
+
+
+## [0.7.0] - 2020-06-01
+### Added
+- `disable_basic_metrics` and `disable_enhanced_metrics` configuration options.
+
+### Changed
+- Major changes in `node_exporter`-like metrics. See [here](https://github.com/percona/rds_exporter#metrics).
```
Comment (AlekSi) [inline CHANGELOG.md:18]: I think that's the best we can do here.

File: CHANGELOG.md (hunk 1 above)
Comment (BupycHuk) [reply CHANGELOG.md]: Maybe description from this PR https://github.com/percona/rds_exporter/pull/33?

**Pull request merged**